test/helpers/DataConverterTests.scala

```diff
@@ -0,0 +1,34 @@
+package helpers
+
+import scala.resources.BaseTestingClass
+
+class DataConverterTests extends BaseTestingClass{
+
+  var dataConverter: DataConverter = _
+
+  before {
+    dataConverter = new DataConverter
+  }
+
+  "imageToBinaryData function" should "return the datatype Array[Byte]" in {
+    val filename = "C:/Users/Admin/IdeaProjects/ScalaFineArtApplication/public/images/favicon.png"
```

Comment:
`"C:/Users/Admin/IdeaProjects/ScalaFineArtApplication/public/images/favicon.png"` in this class is repeated 3 times